### ISSUE TITLE
psnrloss

### ISSUE THREAD

**Issue by mcmingchang** [megvii-model/HINet#37]
训练自己的数据集的时候发现Loss变成nan了，后来发现torch.log((pred - target) ** 2)，当两张图的差异很小时，此处变成了负数。现在换成L1loss重新训练看看

**Comment by ldlshizhu** [comment]
请问L1 loss 训练结果咋样？

**Comment by mcmingchang** [comment]
L1和psnrloss结果差不多，之前的问题是lr设置的问题
